src/components/Timeline/Playbook/Play.js

```diff
@@ -2,9 +2,16 @@ import React from 'react'
 import { connect } from 'react-redux'
 import FlatButtonStyled from 'components/elements/FlatButtonStyled'
 import { fillTemplate, publishEvent } from 'services/playbookService'
+import config from 'config'
+//import appInsights from 'src/appInsights'
+import appInsights from '../../../appInsights'
```

Comment:
remove ../../../ relative path. this should work: 'services/appInsights'

Reply:
In which file was 'services' defined? I cannot find it.

Reply:
And appInsights.js is not under services.